akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2StreamHandling.scala

```diff
@@ -62,10 +62,15 @@ private[http2] trait Http2StreamHandling { self: GraphStageLogic with StageLoggi
   def handleStreamEvent(e: StreamFrameEvent): Unit = {
     updateState(e.streamId, _.handle(e))
   }
+
+  def handleOutgoingCreated(stream: Http2SubStream): Unit =
```

Comment:
Only clients are allowed to create streams, so we didn't track this event before.